clients/python/girder_client/__init__.py

```diff
@@ -650,9 +651,13 @@ def uploadFileToItem(self, itemId, filepath, reference=None, mimeType=None):
         :type reference: str
         :param mimeType: MIME type for the file. Will be guessed if not passed.
         :type mimeType: str or None
+        :param localFile: path with filename used in Girder. Defaults to
+        filepath.
```

Comment:
Line 641 (which is breaking the style test) and 655 (which sphinx will complain about) can both be moved to the preceding line since we now allow 100 characters per line. 👍